.github/README.md

```diff
@@ -35,11 +35,15 @@ Our user guide is at [tabbycat.readthedocs.io](http://tabbycat.readthedocs.io/).
 
 Tabbycat can be deployed in a number of ways. While you can set it up to [run on your own computer](https://tabbycat.readthedocs.io/en/stable/install/local.html) most users will want to run it as a website.
 
-Tabbycat is configured for a '1 click' install to the [Heroku](https://www.heroku.com) web platform. Click this button to begin:
+[Calico](https://calicotab.com/) is Tabbycat's managed hosting service, offering websites for a flat fee and handling all the technical aspects for you. Click this button to deploy to Calico:
```

Comment:
I'd prefer something more like "Calico is a managed hosting service run by one of Tabbycat's developers. For a flat fee, it will host tab websites, automatically manage their setup and performance, and provide ongoing access to the released tab."

(the second half of the sentence I don't feel strongly about, but thought it might be worth spelling out the benefits more precisely)